src/ng/filter/orderBy.js

```diff
@@ -86,6 +86,9 @@
  *
  * **Note:** If you notice numbers not being sorted as expected, make sure they are actually being
  *           saved as numbers and not strings.
+ * **Note:** If null or undefined values are not being sorted as expected, convert them to the empty
```

Comment:
What this note should mention is that `null` is treated as the string `'null'` while sorting. There doesn't need to be a special mention about `undefined`, since it follows the algorithm described above.

I would also refrain from suggesting any work arounds, because there are different ones available, each with its pros and cons (e.g. use an extra predicate to separate null from non-null value, convert  `null` to an empty string or `undefined`, use a custom comparator etc).